api/types/types.proto

```diff
@@ -2224,27 +2224,25 @@ message LockSpecV2 {
 
 // LockTarget lists the attributes of interactions to be disabled.
 message LockTarget {
+    option (gogoproto.goproto_stringer) = false;
+    option (gogoproto.stringer) = false;
+
     // User specifies the name of a Teleport user.
     string User = 1 [ (gogoproto.jsontag) = "user,omitempty" ];
 
     // Role specifies the name of an RBAC role known to the root cluster.
     // In remote clusters, this constraint is evaluated before translating to local roles.
     string Role = 2 [ (gogoproto.jsontag) = "role,omitempty" ];
 
-    // Cluster specifies the name of a leaf Teleport cluster.
-    // This prevents the root cluster's users from connecting to nodes in the leaf cluster
-    // but does not prevent the leaf cluster from heartbeating back to the root cluster.
-    string Cluster = 3 [ (gogoproto.jsontag) = "cluster,omitempty" ];
```

Comment:
Hmm, I don't think you're ever supposed to remove fields from protobuf structs or change their order number. This will lead to incompatibility issues between clients and servers. We usually deprecate the fields that are no longer needed and keep other fields indices the same.

Reply:
This field was added just recently and is not part of any release yet.

Reply:
Ah, gotcha. Sounds good.

Reply:
Since this PR isn't going to be part of 7.0 I opened https://github.com/gravitational/teleport/pull/7661 to address this potential short-lived incompatibility. Please approve if you can.